server/utils/ssr.tsx

```diff
@@ -68,6 +68,7 @@ export const generateMetaComponents = (metaProps: MetaProps) => {
 	} = initialData.communityData;
 
 	const url = `https://${initialData.locationData.hostname}${initialData.locationData.path}`;
+	const isPub = !!initialData.scopeData?.elements.activePub;
```

Comment:
the check on scopeData is needed because, I believe, the test data in ssr.test.tsx is out of date and does not include scopeData. We could fall back to checking the url for the existence of /pub/ for determining if it's a pub if that feels better.

Reply:
The test data is definitely out of date (as are the types for initialData). IMO it feels a little wrong to make changes to the app code to account for bad test data, but this seems fine.

Reply:
Yeah, agreed. I tried to add scopeData to the test data from the storybook but that seemed to cause more problems than it solved, so I punted. I've added a ticket to remind us to update this.